modules/dreamview/frontend/src/components/StatusBar/Electricity.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+
+@observer
+export default class Electricity extends React.Component {
+
+  render() {
+    const {
+      electricityPercentage,
+    } = this.props;
+
+    const percentageString = `${electricityPercentage}%`;
+    const rectWidth = 24 * (electricityPercentage / 100);
+    return (
+    <div className="show-electricity">
+        <div className="battery-icon">
+	    <svg version="1.1">
```

Comment:
indentation is off. Same for all other js files. Please use formatter to fix all lint issues.

Reply:
got it, thank u.